pkg/console/operator/sync_v400.go

```diff
@@ -177,36 +212,44 @@ func SyncConsoleConfig(co *consoleOperator, consoleConfig *configv1.Console, rou
 	if err != nil {
 		return nil, err
 	}
-	logrus.Printf("Updating console.config.openshift.io with hostname: %v \n", host)
-	consoleConfig.Status.ConsoleURL = util.HTTPS(host)
+	httpsHost := util.HTTPS(host)
+	if consoleConfig.Status.ConsoleURL != httpsHost {
```

Comment:
Reduce spam, only log when change happens.

Reply:
+1 to reducing log spam